DOCS-3570: Simplify capture sync page (#3986)
viamrobotics/docs: docs/data-ai/capture-data/capture-sync.md

  - /fleet/data-management/
---

You can use the data management service to capture data from [supported components and services](/data-ai/capture-data/capture-sync/#click-to-see-resources-that-support-data-capture-and-cloud-sync), then sync it to the cloud.
You can also sync data from arbitrary folders on your machine.

## How data capture and data sync works

The data management service writes data from your configured Viam resources to local storage on your edge device and syncs data from the edge device to the cloud:

- The data management service writes captured data to local edge device storage (<file>~/.viam/capture</file> by default).
- The data management service syncs data to the Viam cloud at a configured sync interval using encrypted gRPC calls and deletes it from the disk once synced.

> - You can capture and sync data independently, one can run without the other.

Reviewer: ```suggestion
- You can capture and sync data independently; one can run without the other.
```
[out of diff nit] grammatically incorrect

For more information, see [How sync works](/data-ai/capture-data/advanced/how-sync-works/).

## Configure data capture and sync for individual resources

> 1. Navigate to a configured {{< glossary_tooltip term_id="resource" text="resource" >}} in **Builder** mode in the [Viam app](https://app.viam.com)

Reviewer: ```suggestion
1. Navigate to a configured {{< glossary_tooltip term_id="resource" text="resource" >}} in **Builder** mode in the [Viam app](https://app.viam.com).
```
[nit]

1. Find the **Data capture** section in the resource panel.
1. Click **+ Add method**.

   {{< imgproc src="/tutorials/data-management/add-method.png" alt="Disable data capture toggle" resize="700x" >}}

1. If you see a warning **Capture disabled on data management service**, click **Enable capture on data management service**.
1. Select a **Method** to capture data from.

> 1. Set the capture **frequency** in hertz, for example to `0.1` with `ReadImage` on a camera to capture an image every 10 seconds.

Reviewer: ```suggestion
1. Set the capture **frequency** in hertz, for example to `0.2` with `ReadImage` on a camera to capture an image every 5 seconds.
```
[opt] Might be a more informative example because it's not just powers of 10

## Stop data capture or data sync

### Stop data capture for a specific resource

1. Navigate to the **Data capture** section of your resource's configuration card.
1. Toggle the configured capture method's switch to **Off**.

   {{< imgproc src="/tutorials/data-management/disable-capture-resource.png" alt="Disable data capture toggle" resize="500x" >}}

1. **Save** your config.

### Stop data capture for all resources

1. Navigate to the **Data capture** section of the data management service configuration card.
1. Toggle the **Capturing** switch to **Off**.

   {{< imgproc src="/tutorials/data-management/disable-capture.png" alt="Disable data capture toggle" resize="300x" >}}

1. **Save** your config.

### Disable data sync:

1. Navigate to the **Cloud sync** section of the data management service configuration card.
1. Toggle the **Syncing** switch to **Off**.

   {{< imgproc src="/tutorials/data-management/disable-sync.png" alt="Disable data sync toggle" resize="300x" >}}

1. **Save** your config.

{{< alert title="Tip: More ways to control data capture or sync" color="tip" >}}

For other ways to control data synchronization, see:

> - [conditional sync](/data-ai/capture-data/conditional-sync/)
> - [retention policies](/data-ai/capture-data/advanced/advanced-data-capture-sync/#cloud-data-retention)
> - [sync optimization](/data-ai/capture-data/advanced/advanced-data-capture-sync/#sync-optimization)

Reviewer: ```suggestion
- [Conditional sync](/data-ai/capture-data/conditional-sync/)
- [Retention policies](/data-ai/capture-data/advanced/advanced-data-capture-sync/#cloud-data-retention)
- [Sync optimization](/data-ai/capture-data/advanced/advanced-data-capture-sync/#sync-optimization)
```
[nit]
